### PR TITLE
update lego file for es and tc and fix cap key

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock <sherlockxu@yunify.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Add lego files for es and tc; fix #https://github.com/kubesphere/console/issues/980

### CODE REVIEW

**Comment by Sherlock113** [comment]
/assign @leoendless
Thanks.

File: src/locales/en/lego.js [hunk 1]
```diff
@@ -26,22 +26,22 @@ export default {
...
+  LEGO_UI_NO_RESULTS_FOUND: 'No result found',
   LEGO_UI_LOAD_MORE: 'Load more',
-  LEGO_UI_LOADING_PLACEHOLDER: 'Loading...',
+  LEGO_UI_LOADING_PLACEHOLDER: 'Loading',
   LEGO_UI_ASYNC_NO_RESULT: 'No Result',
   LEGO_UI_CLEAR_ALL: 'Clear All Values',
   LEGO_UI_CLEAR_VALUE: 'Clear Current Value',
   LEGO_UI_PLEASE_SELECT: 'Please select',
-  LEGO_UI_REACH_BOTTOM: 'Has Reached the Bottom',
+  LEGO_UI_REACH_BOTTOM: 'Reach Bottom',
```
Comment (xmfreak) [inline src/locales/en/lego.js:36]: Should I suggest to switch the order? "Bottom reached" sounds better to me.

Comment (Sherlock113) [reply src/locales/en/lego.js]: Thanks. Changed that.

**Pull request merged**